aten/src/ATen/native/mkldnn/xpu/FusionUtils.cpp

```diff
@@ -0,0 +1,79 @@
+#include <ATen/native/mkldnn/xpu/FusionUtils.h>
+
+using namespace at::native::onednn;
+
+namespace at::native::xpu {
+
+onednn::Attr& unary_attr_with_arg(
+    onednn::Attr& attr,
+    std::string_view unary,
+    torch::List<std::optional<at::Scalar>> scalars,
+    std::optional<std::string_view> algorithm) {
+  if (unary == "hardswish") {
+    return attr.append_post_eltwise(
+        1.0f, 1.f / 6.f, 1.f / 2.f, attr.kind_with_hardswish);
+  } else if (unary == "silu") {
+    return attr.append_post_eltwise(1.0f, 1.0f, 0.0f, attr.kind_with_swish);
+  } else if (unary == "leaky_relu") {
+    auto alpha = scalars[0].get().toOptional<at::Scalar>().value().to<float>();
+    return attr.append_post_eltwise(1.0, alpha, 0.f, attr.kind_with_relu);
```

Comment:
following your style.
```suggestion
    return attr.append_post_eltwise(1.0f, alpha, 0.0f, attr.kind_with_relu);
```

Reply:
modified ,thanks for reminding